src/pages/leeszaal/[readingRoomSlug]/index.tsx

```diff
@@ -32,25 +32,34 @@ import {
 	TabLabel,
 	ToggleOption,
 } from '@shared/components';
-import { mock } from '@shared/components/MediaCardList/__mocks__/media-card-list';
 import { WindowSizeContext } from '@shared/context/WindowSizeContext';
 import { useWindowSize } from '@shared/hooks';
 import { SortOrder } from '@shared/types';
 import { createPageTitle } from '@shared/utils';
 
+import { useGetMediaObjects } from '../../../modules/media/hooks/get-media-objects';
```

Comment:
`@media` toevoegen aan tsconfig paths en in `scripts/path-aliases.js` voor eslint en jest configs